### PR TITLE
Update observation docs

### PR DESCRIPTION
**Pull request by tinashechiraya**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 3 sec

File: docs/src/developer/observations.md [hunk 1]
```diff
@@ -265,11 +265,114 @@ This API endpoint allows retrieving, updating, or deleting a specific observatio
...
+    "stoneflies": true,
+    "other_mayflies": true
+  }
+}
+```
+
+### Additional Information
+
+1. **image_0:** Represents a site image. If uploading multiple site images, it would be `image_0`, `image_1`, etc.
+2. For attaching images to pests/groups (e.g., flatworms, stoneflies, other_mayflies), the variable should be named `"pest_0:name_of_pest"`. The value should be the binary file (image), and the corresponding pest/group must be set to `true`.
```
Comment (Jeremy-Prior) [inline docs/src/developer/observations.md]:
```suggestion
2. For attaching images to pests/groups (e.g. flatworms, stoneflies, other_mayflies), the variable should be named `"pest_0:name_of_pest"`. The value should be the binary file (image), and the corresponding pest/group must be set to `true`.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 30 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 34 sec

**Pull request merged**